etc/conf/dev/requirements_base.txt

```diff
@@ -11,3 +11,4 @@ coverage
 pytest-cov
 codecov
 pytest-rerunfailures
+pluggy
```

Comment:
Why is this added here?

Reply:
@majurg  this was the extra requirements which was present in the License-expression Library,and regarding the .whl files it was present in the thirdparty folder of license expression library.I thaught these extra requirements and thirdparty .whl files would be required during configuration after adding license-expression library to the DeltaCode

Reply:
You would only need to add `license_expression` to the `install_requires` in setup.py. 

The .whl files can be added to thirdparty, but these do not need to explicitly added to any requirements files

Reply:
Yes @majurg ,it was my bad. Since .whl file of license_expression is already present in the thirdparty package it would be sufficient to add only `license_expression` in it `install_requires `in the `setup.py`